.github/workflows/release.yml

```diff
@@ -34,4 +38,40 @@ jobs:
         ld_flags: -w -s -extldflags "-static"
         build_tags: -tags netgo
         project_path: cmd/validator
-        extra_files: LICENSE README.md
\ No newline at end of file
+        extra_files: LICENSE README.md
+
+  registery-matrix:
+    name: Push Docker images to Container Registry
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - base: "alpine:3.18"
+            postfix: ""
+          - base: "scratch"
+            postfix: "-scratch"
+          - base: "ubuntu:20.04"
+            postfix: "-ubuntu"
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Log in to the Container registry
+      uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```

Comment:
What's the reason for using a SHA instead of a release version?

Reply:
This is as per the documentation in : https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages
```
# GitHub recommends pinning actions to a commit SHA.
# To get a newer version, you will need to update the SHA.
# You can also reference a tag or branch, but the action may change without warning.
```
I left the same SHA  as specified in the docs.